powersimdata/scenario/analyze.py

```diff
@@ -187,7 +187,7 @@ def get_load_shed(self):
         try:
             # It's either on the server or in our local ScenarioData folder
             load_shed = self._get_data("LOAD_SHED")
-        except OSError:
+        except ValueError:
```

Comment:
I think we would still get an `OSError`from [here](https://github.com/Breakthrough-Energy/PowerSimData/blob/fc4da59642f6c175a4faf061b29b3e821ae48ab9/powersimdata/data_access/data_access.py#L119) if the file doesn't exist on the server (or blob storage, or locally), but based on the slack discussion it sounds like that shouldn't be the case.

Reply:
Done. I dropped the commit where the change happened